projects/testing-library/src/lib/testing-library.ts

```diff
@@ -154,7 +157,12 @@ export async function render<SutType, WrapperType = SutType>(
     }
     renderedPropKeys = Object.keys(newComponentProps);
 
-    fixture.componentRef.injector.get(ChangeDetectorRef).detectChanges();
+    if (
+      properties?.detectChangesOnRender === true ||
+      (properties?.detectChangesOnRender === undefined && detectChangesOnRender === true)
```

Comment:
Do you think we need this check to use the existing config?
I didn't think of it, and I assumed it would only use the property passed to this function.

Reply:
My first intuition is to respect the initial flag from render unless not explicit overwritten in rerender. But my personal goal is to never use any of those two flags - so I have no real opinion here ;) Should I only check for `properties?.detectChangesOnRender === true`?

Reply:
Yea that's my personal goal as well 😅
I would only check for the properties if you don't mind.